docs/usage/self-hosted-configuration.md

```diff
@@ -363,12 +363,6 @@ Before the first commit in a repository, Renovate will:
 The `git` commands are run locally in the cloned repo instead of globally.
 This reduces the chance of unintended consequences with global Git configs on shared systems.
 
-## gitTimeout
```

Comment:
I'm okay to revert the code and docs.

I think you had a reason for the feature in the first place, though:
https://github.com/renovatebot/renovate/issues/13957#issue-1121648932

If that reason is still valid, then maybe re-open the original issue, or create a new one to track the "have a low Git timeout value" concept?